src/sqlfluff/dialects/exasol_keywords.py

```diff
@@ -847,6 +847,13 @@
     "JDBC",
     "DRIVER",
     "CONSUMER",
+    "CONSUMER_GROUP",
+    "STEP",
+    "CPU_WEIGHT",
+    "PRECEDENCE",
+    "GROUP_TEMP_DB_RAM_LIMIT",
+    "USER_TEMP_DB_RAM_LIMIT",
+    "SESSION_TEMP_DB_RAM_LIMIT",
```

Comment:
Should we alphabetically order all these keywords (not just the ones added here) to make it easier to find keywords that are included already and which ones are not?